plugins/cricket-scoreboard/cricket_data_fetcher.py

```
@@ -0,0 +1,476 @@
+    return whole + balls / 6.0
+
+
+def overs_to_balls(overs: float) -> int:
+    """Return the total number of legal balls represented by an overs value."""
+    if overs is None:
+        return 0
+    try:
+        overs = float(overs)
+    except (TypeError, ValueError):
+        return 0
+    whole = int(math.floor(overs))
```

Ruff (0.15.20)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)